README.md

```diff
@@ -117,12 +117,26 @@ For specific toolchain installation, follow instructions in the
   version of the documentation is uploaded daily to
   [doc.riot-os.org](https://doc.riot-os.org).
 
-## Forum
-Do you have a question, want to discuss a new feature, or just want to present
-your latest project using RIOT? Come over to our [forum] and post to your hearts
-content.
+## Asking for help
 
-[forum]: https://forum.riot-os.org
+You can ask for help on the Forum or on Matrix. Please send bug reports and
+feature requests to our [github issue tracker](https://github.com/RIOT-OS/RIOT/issues)
+
+- [forum](https://forum.riot-os.org) is the default place to start asking for
```

Comment:
Forum ... is the default place to ask for help. The Forum provides an archive of prior questions and answers.

Reply:
yes, this comment was about rephrasing.

- [Forum](https://forum.riot-os.org) is the default place to ask for help. Our Forum provides an archive of prior questions and answers.